cmd/plural/deploy.go

```diff
@@ -113,6 +113,8 @@ func build(c *cli.Context) error {
 		installation, err := client.GetInstallation(c.String("only"))
 		if err != nil {
 			return err
+		} else if installation == nil {
```

Comment:
I'd actually expect this to return an error at the client level.  I'll look into whether there's not an api bug there.